pegasus-spark-common/src/main/java/com/xiaomi/infra/pegasus/spark/HDFSFileSystem.java

```diff
@@ -0,0 +1,71 @@
+package com.xiaomi.infra.pegasus.spark;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.URI;
+import java.net.URISyntaxException;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+public class HDFSFileSystem implements RemoteFileSystem {
+
+  public HDFSFileSystem() {}
+
+  private static final Log LOG = LogFactory.getLog(RemoteFileSystem.class);
+
+  public BufferedReader getReader(String filePath) throws PegasusSparkException {
+    try {
+      InputStream inputStream =
+          org.apache.hadoop.fs.FileSystem.get(new URI(filePath), new Configuration())
+              .open(new Path(filePath));
+      BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream));
+      return bufferedReader;
+    } catch (Exception e) {
+      LOG.error("get filePath reader failed from " + filePath, e);
+      throw new PegasusSparkException("get filePath reader failed, [url: " + filePath + "]" + e);
+    }
+  }
+
+  public BufferedWriter getWriter(String filePath) throws PegasusSparkException {
+    try {
+      OutputStreamWriter outputStreamWriter =
+          new OutputStreamWriter(
+              org.apache.hadoop.fs.FileSystem.get(new URI(filePath), new Configuration())
+                  .create(new Path(filePath)));
+      BufferedWriter bufferedWriter = new BufferedWriter(outputStreamWriter);
+      return bufferedWriter;
+    } catch (Exception e) {
+      LOG.error("get filePath writer failed from " + filePath, e);
+      throw new PegasusSparkException("get filePath writer failed, [url: " + filePath + "]" + e);
```

Comment:
The two lines are repeated.
```suggestion
      PegasusSparkException e = new PegasusSparkException("get filePath writer failed, [url: " + filePath + "]" + e);
      log.error(e.getMessage());
      throw e;
```

Reply:
From the previous test, the exception can be ouput the spark log system, so I'm ready remove the `error` log